tests/DotNetLightning.Core.Tests/ClaimReceivedHTLCTests.fs

```diff
@@ -13,12 +13,22 @@ let utf8 = UTF8Encoding()
 let base58 = Encoders.Base58
 let n = Network.TestNet
 
-let alice =
+type Alice = {
+  CommitKey: Key;
+  FinalKey: Key;
+  CommitPubKey: PubKey;
+  finalPubKey: PubKey;
+  R: byte[];
+  RHash: byte[];
+  RevokeCommit: byte[];
+  RevokeCommitHash: byte[]
+}
+let alice: Alice =
```

Comment:
is this change related to CI?

Reply:
oops, i thought it was, but it seems that is just a local problem. i removed this.